httpclient5/src/main/java/org/apache/hc/client5/http/entity/BrotliDecompressingEntity.java

```diff
@@ -32,9 +32,12 @@
  * {@link org.apache.hc.core5.http.io.entity.HttpEntityWrapper} responsible for
  * handling br Content Coded responses.
  *
+ * @deprecated Use Decoder API
```

Comment:
I think it best to use '@link' in '@deprecated' comments. It makes it much easier to follow the bouncing ball 😉